easybuild/easyconfigs/g/gompi/gompi-2015b.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = "Toolchain"
+
+name = 'gompi'
+version = '2015b'
+
+homepage = '(none)'
+description = """GNU Compiler Collection (GCC) based compiler toolchain,
+ including OpenMPI for MPI support."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+gccver = '5.1.0'
+binutilsver = '2.25'
+tcver = '%s-binutils-%s' % (gccver, binutilsver)
+
+# compiler toolchain dependencies
+dependencies = [
+    ('GCC', gccver, '-binutils-%s' % binutilsver),
+    ('binutils', binutilsver, '', ('GCC', tcver)),
+    ('OpenMPI', '1.8.5', '', ('GNU', '%s-%s' % (gccver, binutilsver))),
```

Comment:
OpenMPI 1.8.6 should appear very soon, so this should be bumped if it's available before July 1st...